app/controllers/forest_liana/application_controller.rb

```diff
@@ -96,6 +99,10 @@ def deactivate_count_response
 
     private
 
+    def render_error(exception)
+      render json: { error: exception.message }, status: exception.status
+    end
```

Comment:
Is there a way to make a global util to do this?

Reply:
This is the purpose of this method 😅. It's already available for each controller.